.github/workflows/build_dev_documentation.yml

```
@@ -107,10 +115,24 @@ jobs:
           rm -rf doc-build-dev
           git clone https://HuggingFaceDocBuilder:$WRITE@github.com/huggingface/doc-build-dev
 
-      - name: Make documentation
+      - name: Make MDX documentation
         run: |
           doc-builder build transformers transformers/docs/source --build_dir doc-build-dev --notebook_dir notebooks/transformers_doc --clean --version pr_$PR_NUMBER
 
+      - name: Make HTML documentation
+        run: |
+          cd doc-builder
+          cp -R ../doc-build-dev/transformers/pr_${PR_NUMBER}/* kit/src/routes/
+          cd kit
+
+          npm install
```

```suggestion
          npm ci
```
Given how often things break w/ svelte-kit 😬